vrecord

```diff
@@ -845,6 +845,11 @@ _lookup_choice(){
             VIDEOCODECNAME="H.264"
             MIDDLEOPTIONS+=(-c:v libx264 -crf 18 -pix_fmt yuv420p)
             SUFFIX="" ;;
+        "HuffYUV")
+            VIDEOCODECNAME="HuffYUV"
+            MIDDLEOPTIONS+=(-c:v ffvhuff -pix_fmt yuv422p10le -pred median -context 1)
```

Comment:
Could you add a bit depth condition similar to what is used in Uncompressed Video.

Reply:
You mean the possibility of choose between 8 and 10 bit?

Reply:
Yes, as written, the user may select 8 bit as a decklink input option, but the huffyuv would be forced to 8 bit. I suggest the pix_fmt of the huffyuv encoding match the input (whether the decklink is set to 8 or 10), the same logic is used in the Uncompressed Video encoding.

Reply:
I’ll amend the PR that way.

Reply:
HuffYUV does not support `uyvy422`, I will use `yuv422` instead.

Reply:
`Unknown pixel format requested: yuv422.`

Reply:
Oops, sorry! Planar missing.